test/DebugInfo/Inputs/type-reconstr-names.txt

```diff
@@ -11,3 +11,4 @@ $SSayypXpG ---> Array<Any.Type>
 $S12EyeCandyCore11XPCListenerC14messageHandleryyAA13XPCConnectionV_AA10XPCMessageVxtcvpfiyAF_AHxtcfU_TA.4 ---> Can't resolve type of $S12EyeCandyCore11XPCListenerC14messageHandleryyAA13XPCConnectionV_AA10XPCMessageVxtcvpfiyAF_AHxtcfU_TA.4
 $Ss10CollectionP7ElementQa ---> Can't resolve type of $Ss10CollectionP7ElementQa
 $S12TypeReconstr8patatinoayAA5tinkyVGSgD ---> Optional<patatino>
+$S12TypeReconstr5OuterV3Fooayx_SiGD ---> Can't resolve type of $S12TypeReconstr5OuterV3Fooayx_SiGD
```

Comment:
Why is this invalid?

Reply:
"This is a hack to avoid crashing; a more complete fix requires deeper changes to the type reconstruction logic."

Reply:
We need to do the thing where we pull out all of the generic argument lists from the "parent" type and put them together.